<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>594</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Result_show">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_num">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openbracket">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_closebracket">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:  10px;
    background-color:  rgb(120, 100, 150);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
background-color: rgb(150, 90, 200);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:  10px;
    background-color:  rgb(120, 100, 150);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
background-color: rgb(150, 90, 200);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:  10px;
    background-color:  rgb(120, 100, 150);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
background-color: rgb(150, 90, 200);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:  10px;
    background-color:  rgb(120, 100, 150);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
background-color: rgb(150, 90, 200);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:  10px;
    background-color:  rgb(120, 100, 150);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
background-color: rgb(150, 90, 200);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:  10px;
    background-color:  rgb(120, 100, 150);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
background-color: rgb(150, 90, 200);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:  10px;
    background-color:  rgb(120, 100, 150);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
background-color: rgb(150, 90, 200);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(100, 137, 137);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(158, 188, 108);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:  10px;
    background-color:  rgb(120, 100, 150);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
background-color: rgb(150, 90, 200);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:  10px;
    background-color:  rgb(120, 100, 150);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
background-color: rgb(150, 90, 200);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x_num">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color: rgb(220, 100, 110);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(197, 70, 83);
}</string>
    </property>
    <property name="text">
     <string>x =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ok">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color: rgb(220, 100, 110);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(197, 70, 83);
}</string>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:  10px;
    background-color:  rgb(160, 150, 150);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
background-color: rgb(107, 116, 200);
}</string>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="credit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color: rgb(220, 100, 110);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(197, 70, 83);
}</string>
    </property>
    <property name="text">
     <string>Credit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>430</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:  10px;
    background-color:  rgb(120, 100, 150);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
background-color: rgb(150, 90, 200);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-radius:  10px;
    background-color:  rgb(120, 100, 150);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}
QPushButton:pressed {
background-color: rgb(150, 90, 200);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eq">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>430</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255, 141, 97);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(255, 100, 69);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sub">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255, 141, 97);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(255, 100, 69);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255, 141, 97);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(255, 100, 69);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255, 141, 97);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(255, 100, 69);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sum">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color:  rgb(255, 141, 97);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(255, 100, 69);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deposit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>500</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius:  10px;
    background-color: rgb(220, 100, 110);
    color:  rgb(255, 255, 255);
    font-size:  33px;
}

QPushButton:pressed {
    background-color:  rgb(197, 70, 83);
}</string>
    </property>
    <property name="text">
     <string>Deposit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
